ssz/utils.py

```diff
@@ -26,6 +30,11 @@ def infer_sedes(obj):
             'uint sedes object or uint string needs to be specified for ints',
             obj
         )
+    elif isinstance(obj, Iterable):
+        raise SerializationError(
```

Comment:
Noticing this just now: `SerializationError` is probably not the right choice here. `pyrlp` uses `TypeError`.